src/resource_clients/actor.ts

```diff
@@ -459,3 +462,50 @@ export interface ActorDefinition {
     maxMemoryMbytes?: number;
     usesStandbyMode?: boolean;
 }
+
+type CommonActorPricingInfo = {
+    /** In [0, 1], fraction of pricePerUnitUsd that goes to Apify */
+    apifyMarginPercentage: number;
+    /** When this pricing info record has been created */
+    createdAt: Date;
+    /** Since when is this pricing info record effective for a given Actor */
+    startedAt: Date;
+    notifiedAboutFutureChangeAt?: Date;
+    notifiedAboutChangeAt?: Date;
+    reasonForChange?: string;
```

Comment:
Can we get docs for these properties as well? I think could guess the meaning of `startedAt` or `apifyMarginPercentage`, but I'm honestly getting lost with `notifiedAboutFutureChangeAt` or `reasonForChange`.

Again, I know this is a conversation for a different thread, but `apifyMarginPercentage` / `createdAt` / `startedAt` make sense as "Pricing" properties... the other three, IMO not so much.

Reply:
This is copied more or less as is from the implementation. It doesn't feel right to start writing our own version of the docs here... In a better world, these types would be generated from the OpenAPI spec, but that's a pipe dream, I guess.